token/token.go

```diff
@@ -480,12 +480,12 @@ func (mgm *tokenManager) GenerateUserTokenForIdentity(ctx context.Context, ident
 	return token, nil
 }
 
-// GenerateUnsignedUserAccessToken generates an unsigned OAuth2 user access token for the given identity based on the Keycloak token
-func (mgm *tokenManager) GenerateUnsignedUserAccessToken(ctx context.Context, keycloakAccessToken string, identity *repository.Identity) (*jwt.Token, error) {
+// GenerateUnsignedUserAccessToken generates an unsigned OAuth2 user access token for the given identity based on the OAuth2 token
+func (mgm *tokenManager) GenerateUnsignedUserAccessToken(ctx context.Context, OAuth2AccessToken string, identity *repository.Identity) (*jwt.Token, error) {
```

Comment:
here, `OAuth2AccessToken` should also start with a lowercase, too, eg: `accessToken`